src/Select.js

```diff
@@ -875,11 +871,9 @@ class Select extends React.Component {
 		}
 	}
 
-	onOptionRef(ref, isFocused) {
-		if (isFocused) {
-			this.focused = ref;
-		}
-	}
+       onOptionRef(ref) {
```

Comment:
I think `onFocusedOptionRef` would be clearer, since the function is now only called when the option is focused, and it could really be merged into the `onFocus` handler for options.